hydra-node/src/Hydra/Options.hs

```diff
@@ -650,6 +677,7 @@ toArgs
       ["--network-id", toArgNetworkId networkId]
         <> ["--node-socket", nodeSocket]
         <> ["--cardano-signing-key", cardanoSigningKey]
+        <> ["--contestation-period", show cp]
```

Comment:
If you properly implement the `--contestatio-period` parser to also parse `show`n values, this can be just `show contestationPeriod` and does not require the usage of `UnsafeContestationPeriod` (it's called `Unsafe` for a reason :)